### ISSUE TITLE
Autoupload - android.process.media.. every times comes up..

### ISSUE THREAD

**Issue by Kamero4K** [nextcloud/android#5425]
### Steps to reproduce
1. On my Tablet ( Samsung galaxy Tab A (2019) i got the nextcloud app installed. I configured autoupload.. Wich does work.. A bit.. If the Autoupload is running (is uploading) the message " android.process.media has stopped again. " pops up every few seconds.. if i go into the alarmapp and want to change time, this message imediatly comes up and the alarmapp closes. When autoupload has nothing to do, alarmapp works fine. Oh, and sometimes the Nextcloud app closes too, but not every time.

I tried to clear appdata = no sucsess
I tired reinstall = no sucess
I tried install Dev version = no sucsess
I tried resetting entare tablet = no sucsess
I tried using internal storage = no sucsess
I tried using SD = no sucsess
(Yes newest Version of App and Android)

By the way.. normal data upload works without any issues...
### Expected behaviour
- Tell us what should happen
It should autoupload those files without this errrormessage..

### Environment data
Android version: 9

Device model: 
SM-T515

Stock or customized system:
Its stock.
All availble Updates are installed.

Nextcloud app version:
newest Playstore version 
and
Nextcloud Dev 20200207 
Nextcloud server version:

Tried 17 and 18.
#### Web server error log
```
Insert your webserver log here

```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
- Nothing is written in logs..
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by uusijani** [comment]
I have the same issue on my Samsung Galaxy A9 phone, with "android.process.media has stopped" appearing for virtually every photo taken (though the uploads seem to still go through). My Samsung Galaxy Tab S5e does not have the same issue, nor do I recall seeing this on any other Android devices (from other manufacturers).

I don't use alarmapp (to my knowledge, though I'm not sure which app that is).

**Comment by uusijani** [comment]
Here's output from logcat:

    03-19 16:33:47.099 20119 20119 E AndroidRuntime: FATAL EXCEPTION: main
    03-19 16:33:47.099 20119 20119 E AndroidRuntime: Process: android.process.media, PID: 20119
    03-19 16:33:47.099 20119 20119 E AndroidRuntime: java.lang.RuntimeException: Unable to start receiver com.android.providers.media.MediaScannerReceiver: java.lang.StringIndexOutOfBoundsException: length=1; index=-3
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.app.ActivityThread.handleReceiver(ActivityThread.java:3619)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.app.ActivityThread.access$1300(ActivityThread.java:237)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1803)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:214)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7078)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:964)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime: Caused by: java.lang.StringIndexOutOfBoundsException: length=1; index=-3
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at java.lang.String.substring(String.java:1995)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.providers.media.utils.MPLogger.log(MPLogger.java:118)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.providers.media.MediaProvider.call(MediaProvider.java:5073)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.content.ContentProvider$Transport.call(ContentProvider.java:403)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.content.ContentResolver.call(ContentResolver.java:1763)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.providers.media.MediaScannerReceiver.logToDb(MediaScannerReceiver.java:139)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at com.android.providers.media.MediaScannerReceiver.onReceive(MediaScannerReceiver.java:77)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        at android.app.ActivityThread.handleReceiver(ActivityThread.java:3610)
    03-19 16:33:47.099 20119 20119 E AndroidRuntime:        ... 8 more

The upload process (I think) then dies:

    03-19 16:33:47.164  2044 13653 I ActivityManager: Killing 20133:com.nextcloud.client/u0a0 (adj 100): depends on provider com.android.providers.media/.MediaProvider in dying proc android.process.media (adj 100)

**Comment by tobiasKaminsky** [comment]
Unfortuanetly I cannot see what is causing this.

To clarify, this is only happening when AutoUpload is enabled? Which folder do you configure?

**Comment by uusijani** [comment]
Folders for auto upload? I have Camera twice (`/storage/emulated/0/DCIM/Camera`; once for pictures, the other for videos), OpenCamera (`/storage/emulated/0/DCIM/OpenCamera`) and Screenshots (`/storage/emulated/0/DCIM/Screenshots`).

(I only tried OpenCamera briefly, recently, and have since uninstalled it. The issue at hand was present already back when I first got this phone and installed Nextcloud on it, last July.)

I've always had auto upload enabled until now, so can't say whether it's dependent on that. I could try disabling it next time the issue re-emerges; as dictated by Murphy's law, I was unable to trigger it just now that I intentionally tried to.

Oh, and I am running Beta (so currently 3.11.0 RC4) of the Nextcloud app. Can't remember when I enrolled in.
